rnahybrid.gap

```diff
@@ -41,8 +41,8 @@ algebra alg_pretty implements sig_rnahybrid(alphabet = char, answer = pp) {
     res.botU = Rope("");
     return res;
   }
-  pp ult(<Subsequence qbase, Subsequence tloc>, pp x) {
-    x.x = x.x + 1;
+  pp target_left_flank(<Subsequence tregion, Subsequence mloc>, pp x) {
```

Comment:
is "tregion" a convention? I dont intuitively understand the prefix.

Reply:
Yes :-)
t = target = typically a large sequence like a messenger RNA, 
m = miRNA = typically 22-25nt short RNA sequence

Unfortunately, I used "q" instead of "t" previously - I will rename objects in another PR